packages/core/src/app/common/error/SentryErrorLogger.ts

```diff
@@ -135,6 +135,12 @@ export default class SentryErrorLogger implements ErrorLogger {
     }
 
     private handleBeforeSend: (event: Event, hint?: EventHint) => Event | null = (event, hint) => {
+        if (
+            event.breadcrumbs?.filter((breadcrumb) => breadcrumb.data?.url.includes('convertcart'))
```

Comment:
Is it possible to use and take advantage of https://docs.sentry.io/platforms/javascript/guides/capacitor/configuration/options/#deny-urls

Reply:
Yesterday, the same Sentry issue's URL was https://www.extremekartz.com/checkout, today it is https://mcneelamusic.com/checkout/order-confirmation.
Therefore, I think it is better to identify `convertcart` in breadcrumbs.

Reply:
Found this in github issues https://github.com/getsentry/sentry-javascript/issues/3147, seems similar. It also links a PR for denyUrls.

Reply:
I have added `convertcart` to `denyUrls`.
Maybe we can keep the filter in `beforeSend` as it is the simlar solution mentioned in https://github.com/getsentry/sentry-javascript/issues/3147 .